library/src/main/java/com/github/pwittchen/reactivenetwork/library/network/observing/strategy/PreLollipopNetworkObservingStrategy.java

```diff
@@ -51,7 +51,12 @@
 
         subscriber.add(unsubscribeInUiThread(new Action0() {
           @Override public void call() {
-            context.unregisterReceiver(receiver);
+            //Catch case where the receiver is already unregistered
+            try {
+              context.unregisterReceiver(receiver);
+            } catch (Exception e) {
+              // ignore exception
```

Comment:
Muting exceptions is bad programming practice and we shouldn't do that. There should be a way do allow a user to handle this exception and default implementation of such handling.